install/kubernetes/cilium/values.yaml

```diff
@@ -401,30 +401,30 @@ bpf:
   preallocateMaps: false
   # -- (int) Configure the maximum number of entries in auth map.
   # @default -- `524288`
-  authMapMax: ~
+  authMapMax: 524288
   # -- (int) Configure the maximum number of entries in the TCP connection tracking
   # table.
   # @default -- `524288`
-  ctTcpMax: ~
+  ctTcpMax: 524288
```

Comment:
These map size changes are incorrect. By default, Cilium picks map sizes based on available memory. We deliberately omit the configuration because there's no way to specify one configuration that matches all deployments. Please remove the map reconfiguration from this PR.

(Consequences for this specific change could be that it causes significant packet loss during upgrade)